cmd/headers.go

```diff
@@ -80,6 +80,7 @@ config, see the "copywrite init" command.`,
 		// Append default ignored search patterns (e.g., GitHub Actions workflows)
 		autoSkippedPatterns := []string{
 			".github/workflows/**",
+			"dependabot.yml",
```

Comment:
Is the `dependabot.yml` file at the root of the directory? I recall seeing it in `.github/dependabot.yml` previously

Reply:
whoops, you're totally right! will fix